posthog/api/team.py

```diff
@@ -208,6 +210,18 @@ def validate_session_recording_network_payload_capture_config(self, value) -> Di
 
         return value
 
+    def validate_session_recording_config(self, value) -> Dict | None:
+        if value is None:
+            return None
+
+        if not isinstance(value, Dict):
+            raise exceptions.ValidationError("Must provide a dictionary or None.")
+
+        if not all(key in ["record_canvas"] for key in value.keys()):
+            raise exceptions.ValidationError("Must provide a dictionary with only 'record_canvas' key.")
```

Comment:
Maybe it should not be a dictionary then?

Reply:
Ah, I'd forgotten my pre-xmas good citizen push that we should move session replay config into one place so we can stop adding more nad more columns to the team model

Maybe we should get this in and then I use it for the config in https://github.com/PostHog/posthog/pull/19887